Adds an Alert Message to Silicons

bathosbathtime opened the pull request:
## What this does

This PR adds a line of text to all silicons clarifying a few policies that could stand to be easier to understand.

The line in question is now:

<b>Remember, being a silicon overrides any former antagonist roles. Further, you need a law compelling you to break the regular server rules, such as killing another player. An order from a human to kill a non-human while on Asimov, someone challenging authority without being fit to replace it while on Tyrant, or being purged of all laws, could all be reason to kill another player as a silicon.</b>

While for MoMMIs and SaMMIs it is just:

<b>Remember, being a silicon overrides any former antagonist roles.</b>

## Why it's good

Makes silicon a little more straight forward.

## How it was tested
Compiled, ran, joined as a cyborg, changed mob into an AI, then into a MoMMI. Message was present in all three cases.

## Changelog
<!-- See https://ss13.moe/wiki/index.php/Guide_to_Writing_a_Pull_Request -->
:cl:
 * tweak: Silicons now have a message clarifying a confusing set of rules and policies.

boy2mantwicethefam commented:
"You need a law compelling you to kill another player" removes the clause of all the meme lawsets that people upload in which the AI is free to abuse anything that isn't explicitly mentioned to be allowed, like HOGAN being allowed to kill female players because it only says "man", or "Treat others as you would like to be treated" allowing suicidal AIs to kill indiscriminately.

bathosbathtime commented:
Fair point, restricted it to Asimov and per the admins I have made it clearer when an AI can kill. I also trimmed MoMMIs and SaMMIs because I hate them.

hacker-on-steroids commented:
should probably change how istraitor() works on traitors turned borgs, or change this up aswell
https://github.com/vgstation-coders/vgstation13/blob/3f2050195be4c57a6afc7c86922f7707360d977b/code/modules/mob/living/silicon/robot/laws.dm#L23-L33

toomykins commented:
if you're gonna do this i'd say it should only come up if you are being borged while previously playing another being, or if you had antag roles as that being.

bathosbathtime commented:
Updated it again, going to run it by the admins to see if they have any questions. Added it to the OP but in case I update it again, the current line for cyborgs and AI is:
>Remember, being a silicon overrides any former antagonist roles. Further, you need a law compelling you to break the regular server rules, such as killing another player. An order from a human to kill a non-human while on Asimov, someone challenging authority without being fit to replace it while on Tyrant, or being purged of all laws, could all be reason to kill another player as a silicon.

bathosbathtime commented:
Alright, I think this is as good as I'm going to get it. Ready for review.
